i18n/en.json

```diff
@@ -1508,6 +1508,10 @@
     "id": "api.license.request_trial_license.app_error",
     "translation": "Unable to get a trial license, please try again or contact with support@mattermost.com."
   },
+  {
+    "id": "api.license.request_trial_license.no-site-url.app_error",
+    "translation": "Unable to request a trial license, please configure your SiteURL first in the web server section of the mattermost configuration."
```

Comment:
Can we get approval of this error message from PM or technical writer?

Reply:
I guess so

Reply:
@justinegeffen can you review this text?